src/graphql/schema/buildInputObject.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  GraphQLInputFieldConfigMap,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLString,
+  Thunk,
+} from 'graphql';
+
+const buildInputObject = (name: string, fieldTypes: Thunk<GraphQLInputFieldConfigMap>): GraphQLInputObjectType => {
+  return new GraphQLInputObjectType({
+    name: `${name}_where`,
+    fields: {
+      ...fieldTypes,
+      OR: {
+        type: new GraphQLList(new GraphQLInputObjectType({
+          name: `${name}_where_or`,
+          fields: fieldTypes,
+        })),
+      },
+      AND: {
+        type: new GraphQLList(new GraphQLInputObjectType({
+          name: `${name}_where_and`,
+          fields: fieldTypes,
+        })),
+      },
+      page: { type: GraphQLInt },
+      limit: { type: GraphQLInt },
+      sort: { type: GraphQLString },
```

Comment:
I'm confused about what this is. It's called `buildInputObject` but the name of the object type is `_where`. Then, the properties of the object type start to look like `where` properties like `OR`, `AND`, etc - but then `page` and `limit`, etc. are not `where` properties.

Maybe we should talk through this tomorrow?